### PR TITLE
Fix mapping for geographic subject authority.

### LINKED ISSUE #2193: Fix mapping for geographic subject authority
For druid:hv324dj9498

```
Original node:
<subject altRepGroup="01" authority="lcsh">
  <geographic>M&#xE1;lia Site (Greece)</geographic>
</subject>
Best guess roundtripped node:
<subject altRepGroup="1" authority="lcsh">
  <geographic authority="lcsh">Mália Site (Greece)</geographic>
</subject>
```

Notice the extra authority attribute on `<geographic>`.

### PR DESCRIPTION
**Pull request by justinlittman**
closes #2193

## Why was this change made?
It's complicated.

## How was this change tested?
Unit, sdr-deploy

## Which documentation and/or configurations were updated?
NA

### CODE REVIEW

File: spec/services/cocina/mapping/descriptive/mods/subject_geographic_spec.rb [hunk 1]
```diff
@@ -217,6 +217,53 @@
...
+                  ]
+                }
+              ]
+            }
+          ]
+        }
+      end
+    end
+  end
+
```
Comment (arcadiafalcone) [inline spec/services/cocina/mapping/descriptive/mods/subject_geographic_spec.rb:266]: Parallel values should have the same content - the Latin subject should have the topic subdivision as well.

Comment (justinlittman) [reply spec/services/cocina/mapping/descriptive/mods/subject_geographic_spec.rb:299]: It doesn't in `druid:hv324dj9498`. Change the example or leave true to the original?

**Comment by justinlittman** [comment]
Before:
```
Status (n=100000):
  Success:   98483 (98.483%)
  Different: 1007 (1.007%)
  To Cocina error:     0 (0.0%)
  To Fedora error:     0 (0.0%)
  Missing:     510 (0.51%)
```

After:
```
Status (n=100000):
  Success:   98485 (98.485%)
  Different: 1005 (1.005%)
  To Cocina error:     0 (0.0%)
  To Fedora error:     0 (0.0%)
  Missing:     510 (0.51%)
```

File: spec/services/cocina/mapping/descriptive/mods/subject_geographic_spec.rb (hunk 1 above)
Comment (arcadiafalcone) [reply spec/services/cocina/mapping/descriptive/mods/subject_geographic_spec.rb:299]: Change the example and I'll remediate the original.

Comment (justinlittman) [reply spec/services/cocina/mapping/descriptive/mods/subject_geographic_spec.rb:308]: Done @arcadiafalcone

**Review by arcadiafalcone — APPROVED** [review]

**Comment by justinlittman** [comment]
Requires Infra dev review.

**Review by jcoyne — APPROVED** [review]

**Pull request merged**